server/api/controllers/login_controller.go

```diff
@@ -22,8 +26,25 @@ func (server *Server) Login(c *gin.Context) {
 		utils.ERROR(w, http.StatusUnprocessableEntity, err)
 		return
 	}
+		
+	type PostBody struct {
```

Comment:
This way of parsing the JSON body is so gross I feel like the worst developer in the world for writing this but I'm also bad at go.  Data is passed in like this and we want to parse the data key in the JSON body whose value is a string. Help @rishabluthra 

![image](https://user-images.githubusercontent.com/46202699/101270606-91e7e980-3748-11eb-88ca-1247788e9dfd.png)